backend/internal/storage/postgres/feedback_comments.go

```diff
@@ -0,0 +1,70 @@
+package postgres
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/CamPlume1/khoury-classroom/internal/models"
+	"github.com/jackc/pgx/v5"
+)
+
+// gets all feedback comments on a student work
+func (db *DB) GetFeedbackOnWork(ctx context.Context, studentWorkID int) ([]models.PRReviewCommentResponse, error) {
+	query := `SELECT student_work_id, rubric_item_id, github_username, file_path, file_line, fc.created_at, assignment_outline_id, point_value, explanation
```

Comment:
having the abbreviated name for rubric_items but not for feedback_comment is odd